generator/genutils/kube_liike_versions.go

```diff
@@ -0,0 +1,41 @@
+package genutils
```

Comment:
Typo in this file name (`liike -> like`)

Reply:
Fixed in commit https://github.com/devfile/api/pull/273/commits/b65067d128173c58b17a9c2d908743e95e901b7e